bin/runtime/src/lib.rs

```diff
@@ -575,26 +574,19 @@ impl pallet_multisig::Config for Runtime {
     type WeightInfo = pallet_multisig::weights::SubstrateWeight<Runtime>;
 }
 
-// We do not burn any money within treasury.
-pub const TREASURY_BURN: u32 = 0;
-// The percentage of the amount of the proposal that the proposer should deposit.
-// We agreed on non-progressive deposit.
-pub const TREASURY_PROPOSAL_BOND: u32 = 0;
-// The proposer should deposit max{`TREASURY_PROPOSAL_BOND`% of the proposal value, 100 AZERO}.
-pub const TREASURY_MINIMUM_BOND: Balance = 100_000_000_000 * TOKEN;
-pub const TREASURY_MAXIMUM_BOND: Balance = 100_000_000_000 * TOKEN;
-// Every 4 hours we implement accepted proposals.
-pub const TREASURY_SPEND_PERIOD: BlockNumber = 4 * BLOCKS_PER_HOUR;
-// We allow at most 20 approvals in the queue at once.
-pub const TREASURY_MAX_APPROVALS: u32 = 20;
-
 parameter_types! {
-    pub const Burn: Permill = Permill::from_percent(TREASURY_BURN);
-    pub const ProposalBond: Permill = Permill::from_percent(TREASURY_PROPOSAL_BOND);
-    pub const ProposalBondMinimum: Balance = TREASURY_MINIMUM_BOND;
-    pub const ProposalBondMaximum: Balance = TREASURY_MAXIMUM_BOND;
-    pub const MaxApprovals: u32 = TREASURY_MAX_APPROVALS;
-    pub const SpendPeriod: BlockNumber = TREASURY_SPEND_PERIOD;
+    // We do not burn any money within treasury.
+    pub const Burn: Permill = Permill::from_percent(0);
+    // The fraction of the proposal that the proposer should deposit.
+    // We agreed on non-progressive deposit.
+    pub const ProposalBond: Permill = Permill::from_percent(0);
+    // The minimal deposit for proposal. This value effectively disables treasury.
+    pub const ProposalBondMinimum: Balance = 100_000_000_000 * TOKEN;
+    pub const ProposalBondMaximum: Balance = 100_000_000_000 * TOKEN;
```

Comment:
let's be pedantic locos for 100%: pls add short description for this upperbound

Reply:
Sometimes you scare me... :P Done